fabric-item-materials-v0/src/main/java/net/fabricmc/fabric/api/itemmaterial/FabricArmorMaterial.java

```diff
@@ -0,0 +1,71 @@
+package net.fabricmc.fabric.api.itemmaterial;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.impl.itemmaterial.ArmorMaterialsExtensions;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.item.ArmorMaterial;
+import net.minecraft.item.ArmorMaterials;
+import net.minecraft.recipe.Ingredient;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.Lazy;
+
+import java.util.function.Supplier;
+
+public class FabricArmorMaterial implements ArmorMaterial {
```

Comment:
Do we need these in the API? They're nice helpers, but do we need them?

Reply:
Nope! 100% not needed.